.github/workflows/pull-release.yml

```diff
@@ -132,6 +138,7 @@ jobs:
           gateway_service_version: ${{ steps.gateway-service.outputs.tag }}
           entity_service_version: ${{ steps.entity-service.outputs.tag }}
           config_service_version: ${{ steps.config-service.outputs.tag }}
+          hypertrace_service_version: ${{ steps.hypertrace-service.outputs.tag }}
 
       - name: update release notes
```

Comment:
can we pass Hyeprtrace service version to `release_notes` step as well?

Also add hypertrace-service in compatibility matrix file. Maybe, we can hold it of for release notes but I am fine with adding it there too.

Reply:
I am purposefully skipping it from release notes and compatibility matrix, till the time it is disabled by default

Reply:
got it.

Reply:
I think it is enabled `condition: merge-query-services.enabled'`, but it makes sense after we have `macro-repo`.